src/main/java/com/github/bbooong/bangumall/payment/application/PaymentService.java

```diff
@@ -0,0 +1,57 @@
+package com.github.bbooong.bangumall.payment.application;
+
+import com.github.bbooong.bangumall.payment.application.dto.PaymentCreateRequest;
+import com.github.bbooong.bangumall.payment.application.dto.PaymentInfoResponse;
+import com.github.bbooong.bangumall.payment.domain.OrderClient;
+import com.github.bbooong.bangumall.payment.domain.OrderInfo;
+import com.github.bbooong.bangumall.payment.domain.Payment;
+import com.github.bbooong.bangumall.payment.domain.PaymentClient;
+import com.github.bbooong.bangumall.payment.domain.PaymentRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class PaymentService {
+
+    private final PaymentClient paymentClient;
```

Comment:
```suggestion
    private final BanguPayClient bangupayClient;
```

Payment 접두어를 공유해서 헷갈리는데 외부 클라이언트를 나타내는 이름으로 바꾸는건 어떤가요?